zipline/pipeline/loaders/blaze/core.py

```diff
@@ -1011,12 +1011,47 @@ def last_in_date_group(df, reindex, have_sids=have_sids):
                     )
                 else:
                     last_in_group = last_in_group.reindex(dates)
+            # Unstack will fill all missing values with NaN; we need to fix
+            # this for all types that are not float.
+            if not df.empty:
+                for column in columns:
+                    if isinstance(column.missing_value, NotSpecifiedType):
```

Comment:
I don't think this can ever happen.  By the time we have a loadable column, we should always have a concrete missing value.